ProjektWPF/ProjektWPF/Czas/Data.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjektWPF.Czas
+{
+    public class Data
+    {
+        public string[] DzienTygodnia = {"poniedziałek", "wtorek", "środa", "czwartek", "piątek", "sobota", "niedziela"};
```

Comment:
dlaczego pole DzienTygodnia jest jako public?
Ta tablica powinna byc nie zmienna czyli const

Reply:
Rzeczywiście można zrobić const, bo tablica nie będzie ulegała zmianom, jednak jest ona public po to, aby etykieta z bieżącym czasem mogła czerpać dzień tygodnia po polsku, który domyślnie jest po angielsku.